jams/eval.py

```diff
@@ -32,6 +32,9 @@ def validate_annotation(ann, namespace):
     '''Validate that the annotation has the correct namespace,
```

Comment:
Should we be concerned about potential confusion between `eval.validate_annotation()` and `annotation.validate()` ?